### PR TITLE
ADAPT-000: Added some misc repo files.

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

### CODE REVIEW

**Comment by sherakama** [comment]
@nplowman & @mattanglin ping.

**Review by nplowman — APPROVED** [review]
Sorry - didn't realize this was waiting on a review from me.
Everything looks good to me.
I imagine we can't test out the version auto-tagging without actually merging a PR to main. But if there are any issues I can make tweaks to it when I set up the npm publishing.

**Pull request merged**
